build/test_encrypt.py

```diff
@@ -1,47 +1,42 @@
 """This is unit test for text file Encryption 
 and Decryption.
 """
-    
-import unittest
-import os
 from binascii import Error
 import cryptography
 from cryptography.fernet import Fernet
+import unittest
 
-class TestEncryptDecrypt(unittest.TestCase):
+class TestEncryption(unittest.TestCase):
     def setUp(self):
-        self.key_path = "D:\zip file version\client-server-network-combatzone-2\static\crypto.key"
-        self.file_path = "D:\zip file version\client-server-network-combatzone-2\static\serdes.dat"
-
-        # Generate and save a key
-        key = Fernet.generate_key()
-        with open(self.key_path, "wb") as crypto_file:
-            crypto_file.write(key)
-
-    def test_encrypt_decrypt(self):
+        # Generate and save the key
+        write_key()
```

Comment:
we better take this part out, because it won't work without complete "key generation" function which is removed in this version. 
I think we **shouldn't generate** new keys for test case, what do you think? Because even if we keep "Fernet.generate_key" function here, that will generate **new keys each time** we run the test, and when we run the test that will create one key while encrypting the file, and another new key while decrypting the file which will end up with conflict and the code won't run.